sdk/formrecognizer/tests.yml

```diff
@@ -4,4 +4,19 @@ extends:
   template: ../../eng/pipelines/templates/stages/archetype-sdk-tests.yml
   parameters:
     ServiceDirectory: formrecognizer
-    TimeoutInMinutes: 90
+    TimeoutInMinutes: 240
+    CloudConfig:
+      Public:
```

Comment:
could you add `Location: 'canadacentral'` here as it got deleted from `test-resources.json`?

Reply:
@Luyunmt

Reply:
OK